artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/TransportConstants.java

```diff
@@ -125,6 +125,10 @@
     */
    public static final String NIO_REMOTING_THREADS_PROPNAME = "nioRemotingThreads";
 
+   public static final String NETTY_WRITE_BUFFER_LOW_WATER_MARK_PROPNAME = "nettyWriteBufferLowWaterMark";
```

Comment:
If you are exposing the parameter to users, can you rename it as bufferLowWatermark
and bufferHighWaternark?

It's more independent from Netty and more user friendly.